client/reader/recommended-sites/recommended-site.tsx

```diff
@@ -99,6 +91,7 @@ const RecommendedSite = ( {
 }: RecommendedSiteProps ) => {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
+	const railcarId = railcar?.railcar as string;
```

Comment:
There should be no need to use `as` here. Because you are assuming that `railcar?.railcar` is a string, this means that we should expect the `railcar` object to always be there (i.e. make it required), and it should have a required `railcar` prop of type `string`. 🤔 
Unless the railcar object may actually be undefined. Then this should be handled accordingly too, without the need to override the type. I can help you look at this, if you'd like to.